File: class/class5.1/MegaData.cpp
#include "MegaData.h"
#include <algorithm>

MegaData::MegaData() {
    reset();
}

void MegaData::reset() {
    std::fill(std::begin(smallArray), std::end(smallArray), 42.0f);
    std::fill(std::begin(bigArray), std::end(bigArray), 42.0);
}

File: home/home5.1/MegaData.swift
import Foundation

class MegaData {
    var smallArray: [Float]
    var bigArray: [Double]

    init() {
        smallArray = Array(repeating: 42.0, count: 1024)
        bigArray = Array(repeating: 42.0, count: 1024 * 1024)
    }

    func reset() {
        smallArray = Array(repeating: 42.0, count: 1024)
        bigArray = Array(repeating: 42.0, count: 1024 * 1024)
    }
}
